src/components/TagModal.vue

```diff
@@ -173,7 +173,10 @@ export default {
 		async createTag(event) {
 			this.editing = true
 			const displayName = event.target.querySelector('input[type=text]').value
-
+			if(this.$store.getters.getTags.some(tag=>tag.imapLabel === "$"+displayName)){
```

Comment:
```suggestion
			if(this.$store.getters.getTags.some(tag => tag.displayName === displayName)){
```
Tags have a display name already so you can omit the conversion. This also fixes linter issues.